Dockerfile

```diff
@@ -4,7 +4,8 @@ LABEL maintainer="https://github.com/tootsuite/mastodon" \
       description="A GNU Social-compatible microblogging server"
 
 ENV RAILS_ENV=production \
-    NODE_ENV=production
+    NODE_ENV=production \
+    SMTP_CA_FILE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
We're not setting any other variables by default in the `Dockerfile` (the two above are the exception, because they have to be set _before_ dotenv runs).

Can we take this out of here, and instead add it to the example dotenv file for production?